### DISCUSSION TITLE
Support for angular 15

### DISCUSSION THREAD

**Discussion by hgrewa** [ngxs/store#1940] [Ideas] [answered]
Do we know if adding support for angular 15 is in the pipeline? As of now, this library can not be installed on ng 15 app.
Has anybody found a workaround?

-----------
ERR! Could not resolve dependency:
npm ERR! peer @angular/core@">=6.1.0 <15.0.0" from @ngxs/store@3.7.5
------------

**Comment by asirko** [comment]
There is a commit 2 days ago called "feat: add support for Angular 15".

The question is more "what is the release plan?"

**Comment by markwhitfeld** [comment]
Planning to do the release later today.
I was without power yesterday and had a family member in hospital just before that.

**Comment by markwhitfeld** [comment, marked as answer]
New version `v3.7.6` has been released with Angular 15 support and a number of other improvements 🎉 
Discussion: https://github.com/ngxs/store/discussions/1944
Release: https://github.com/ngxs/store/releases/tag/v3.7.6
